client/lib/pipelineUtils.ts

```diff
@@ -66,7 +66,10 @@ const transformData = (arr: Company[]) => {
   });
 
   // is this right?
-  columns[Status.Applied].sort((o) => Date(o.submitted_at));
+  columns[Status.Applied].sort(
```

Comment:
Is the issue pushing into a const? so should do this on arr before pushing its elements? i.e. on line 55 instead of after?